js/app.js

```diff
@@ -646,6 +640,38 @@ define([
 				});
 				return resolvingPromise;
 			};
+			self.getConceptIdentifierList = function() {
+        return sharedState.selectedConcepts().map(item => item.concept.CONCEPT_ID);
+      };
+			self.resolveInclusionConceptCount = function(conceptSetId) {
+        if (self.currentConceptSet() && self.currentConceptSet().id == conceptSetId) {
+			    return $.Deferred().resolve();
+        }
+			  self.resolvingConceptSetExpression(true);
+        const conceptSetExpression = {items: sharedState.selectedConcepts(),};
+			  return conceptSetApi.includedConceptSetCount(ko.toJSON(conceptSetExpression))
+          .then(count => {
+            self.resolvingConceptSetExpression(false);
+            self.conceptSetInclusionCount(count);
+           }, () => {
+            self.resolvingConceptSetExpression(false);
+            document.location = '#/configure';
+          });
+      };
+			self.resolveIncludedConcepts = function(conceptSetId) {
+        if (self.currentConceptSet() && self.currentConceptSet().id == conceptSetId && self.conceptSetInclusionIdentifiers().length > 0) {
+          return $.Deferred().resolve();
```

Comment:
Why not Promise.resolve()?

Reply:
I think this is just a situation where we have legacy code using jquery based promises and we have new code introduced which uses native promises.  We should think about an upgarde to jquery 3 which follows the same semantics as the published standard of promises.